### DISCUSSION TITLE
react-hook-form Checkbox inside a Field.Root

### DISCUSSION THREAD

**Discussion by jerome-diver** [chakra-ui/chakra-ui#9265] [Q&A] [answered]
# How to manage a Checkbox with react-hook-form:
- To do a
`setValue("enable", true)`
and show checked box ?

- And a 
`reset()`
to show uncheck box ?

I'm using a 
`const { setValue, register, handleSubmit, reset } = useForm<Page>({ defaultValues: {} })`

**Comment by segunadebayo** [comment, marked as answer]
I've just pushed some updates to the checkbox docs. You can also see it here:
https://github.com/chakra-ui/chakra-ui/commit/82712fbe9d487a7a2a93d1f9cfa6921b126d0593
